apps/i18n/netsim/az_az.json

```diff
@@ -52,6 +52,8 @@
   "lobbyStatusWaitingForYou": "Waiting for you...",
   "logBrowserButton": "Log Browser",
   "logBrowserHeader": "All Router Logs",
+  "logBrowserHeader_all": "All Router Logs",
```

Comment:
Just curious, was there an automated tool for making all of these localization changes based on the initial English one?